sesman/session.c

```diff
@@ -89,6 +89,37 @@ dumpItemsToString(struct list *self, char *outstr, int len)
     return outstr ;
 }
 
+/* find the last occurrence of ch in str limited by n */
+static int
+last_index_of(const char *str, char ch, int n)
```

Comment:
This function should be placed `common/` because it has nothing sesman-specific.
It looks general helper function handling string.